.changelog/591.txt

```diff
@@ -0,0 +1 @@
+update hcp_vault_plugin resource test
```

Comment:
Looks like this may not be formatted properly. It should look something like this

`
```release-note:improvement
update hcp_vault_plugin resource test
```
`

Reply:
thanks, updated the msg